dist/index.js

```diff
@@ -294,6 +294,18 @@ function removeIgnoredAttributes(taskDef) {
   return taskDef;
 }
 
+function maintainAppMeshConfiguration(taskDef) {
+    if ('proxyConfiguration' in taskDef && taskDef.proxyConfiguration.type == 'APPMESH' && taskDef.proxyConfiguration.properties.length > 0) {
+        taskDef.proxyConfiguration.properties.forEach((value, index, arr) => {
+            if (!('value' in value)) {
```

Comment:
If we send empty property name instead of a value then also we end up getting the same error. So can we modify this logic such that it will check for both name and value is empty or not?  if name is empty then make both name and value empty. How does it sound to you?

Reply:
I can add code that ensures `name` is also present. The reason I didn't is I didn't think it likely that someone would provide a value without a key. I'll push up a change for this shortly.